css/sections/footer.postcss

```diff
@@ -0,0 +1,80 @@
+/*----------------------------------------*\
+  FOOTER
+\*----------------------------------------*/
+
+.footer-information {
+  @apply bg-secondary-600;
```

Comment:
isn't our order from inside out and with boxmodel? 
so: 
1 Gridinfo
2 Paddings
3 Background

?